### PR TITLE
fixPurty script that will make purescript code pretty

### PR DESCRIPTION
**Pull request by shmish111**
Michael I've included you in this PR since you are the de-facto "nix guy" now, sorry.

### CODE REVIEW

File: purty/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+```
+  - socks-0.6.0
+  - connection-0.3.0
+```
+This is required to make sure `stack2nix` uses the correct version of these libraries.
+
+Now you can run:
+```
+nix-shell -p stack2nix
+stack2nix . > packages.nix
```
Comment (michaelpj) [inline purty/README.md]: This is pretty underdetermined. The one for the main packages pins everything so it's remotely reproducible. I guess it matters less for this random package so not a big deal.

File: purty/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Purty
+
+Currently Purty is a little difficult to build with nix for two reasons:
```
Comment (michaelpj) [inline purty/README.md]: I don't suppose we can get it from nixpkgs, can we? I'm seriously considering adding a `nixpkgs-unstable` pin for a few tools that we use for the docs, is purty in `nixpkgs-unstable`?

File: purty/README.md (hunk 1 above)
Comment (shmish111) [reply purty/README.md]: Yeah, it's an offline process anyway, I figured it doesn't really matter that much. When stackage upgrades `network` we should be able to get rid of all this and add purty to our `stack.yaml`

File: purty/README.md (hunk 2 above)
Comment (shmish111) [reply purty/README.md]: no, it will likely be quite a while before a stable purty is in nixpkgs, it's currently quite a young and fast-moving project.

**Comment by michaelpj** [comment]
Wasn't paying enough attention for one nit: I've been shoving misc nix stuff into `nix`, so probably the `purty` folder should go in there. NBD, and I wouldn't bother moving it now.

**Pull request merged**